### ISSUE TITLE
Autoload: file_exists() serves no purpose

### ISSUE THREAD

**Issue by sanderdekroon** [MultiSafepay/WooCommerce#4]
Hi!

In your Autoload.php you are using realpath() to fully resolve the absolute path to a file. After this you are checking the path with file_exists() function. But the PHP documentation states:

> realpath() returns FALSE on failure, e.g. if the file does not exist.

So file_exists() serves no purpose here.
https://github.com/MultiSafepay/WooCommerce/blob/9f6940bddfc9d5bb6dc440c5ec6c70b76b998cbf/wp-content/plugins/multisafepay/includes/classes/Autoload.php#L41-L51

Also, on line 42 you are also checking the existence of files with a relative path. But this is causing a lot of warnings on systems where open_basedir restriction are in effect:
`PHP message: PHP Warning:  file_exists(): open_basedir restriction in effect. File(MultiSafepay/api/Minify_HTML) is not within the allowed path(s): (/var/hpwsites/xxxx:/usr/share/pear/) in /var/hpwsites/xxxx/website/html/webroot/wp-content/plugins/multisafepay/includes/classes/Autoload.php on line 42`

Wouldn't it be better to also include an absolute path, instead of a relative path?

**Comment by jasper-multisafepay** [comment]
@sanderdekroon 
You're right on the first part and have created an internal ticket for this.
Will also discuss the second part with the plugin developer to take this into account.
Thanks for reporting this.
